### ISSUE TITLE
Private routes, params and componentProps

### ISSUE THREAD

**Issue by aaronksaunders** [ionic-team/stencil-router#38]
when looking at the tests you have here

https://github.com/ionic-team/stencil-router/blob/master/src/components/__tests__/test-app.tsx#L5

and using code similar to what you have provided for the `PrivateRoute` when I use this code, url params do not get passed down, only specific component props get passed through?

So if I have something like this...

```
<stencil-router>
  <PrivateRoute path="/" component="app-home" exact={true} />
  <PrivateRoute url="/profile/:name" component="app-profile" />
  <stencil-route url="/login" component="app-login" exact={true} />
</stencil-router>
```

i am not getting the value for `:name` passed through...

if I change the code to `PrivateRoute` to the following, then it appears to work
```
const PrivateRoute = ({ component, ...props}: { [key: string]: any}) => (
  <stencil-route {...props} routeRender={
    (props: { [key: string]: any}) => {
      debugger;
      console.log(props)
      console.log(component)
      if (firebaseAPI.getUser()) {
        const Component = component;
        // why do they use componentProps and not just pass thru
        // the props?
        // return <Component {...props.componentProps}></Component>;
        return <Component {...props}></Component>;
      }
      return <stencil-router-redirect url="/login"></stencil-router-redirect>
    }
  }/>
)
```

**Comment by jthoms1** [comment]
Good catch,  I will update the component to allow passing of both props and componentProps. Thank you!

**Comment by jthoms1** [comment]
This was fixed with commit  3465fc1.
